### PR TITLE
doc: fix layout of OperatorHub doc

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
Just fix the layout for the doc: https://pkg.go.dev/github.com/operator-framework/api@master/pkg/validation/internal#OperatorHubValidator

### CODE REVIEW

File: pkg/validation/internal/operatorhub.go [hunk 1]
```diff
@@ -22,33 +22,55 @@ import (
...
+// * Basic Install
+//
+// * Seamless Upgrades
+//
+// * Full Lifecycle
+//
+// * Deep Insights
+//
+// * Auto Pilot
+//
```
Comment (tlwu2013) [inline pkg/validation/internal/operatorhub.go:37]: +1 on this added spacing since it helps separate "capabilities" above and the "categories" below.

**Review by tlwu2013 — COMMENTED** [review]
I am not sure if it's just me but adding all these extra spacings makes me feel it's a bit harder to read through the comments.

I do like some extra spacings to better separate the zoning though. So perhaps the rule could be we add spacings between every `-` but no extra spacings between `*` so things are that not wide spread but still provides some zoning? 

What do you think?

**Comment by camilamacedo86** [comment]
Hi @tlwu2013, 

> I am not sure if it's just me but adding all these extra spacings makes me feel it's a bit harder to read through the comments.

The changes here are required to fix the layout for its docs to be generated properly. We ought to not ask for a look in the code and instead provide the link: https://pkg.go.dev/github.com/operator-framework/api@master/pkg/validation/internal#OperatorHubValidator 

> I do like some extra spacings to better separate the zoning though. So perhaps the rule could be we add spacings between every - but no extra spacings between * so things are that not wide spread but still provides some zoning?

The spaces are to allow breaking the lines in the golang docs generated for the API. adopted - instead of * will not solve this need.

**Comment by tlwu2013** [comment]
thanks for the clarification, sgtm @camilamacedo86

File: pkg/validation/internal/operatorhub.go [hunk 2]
```diff
@@ -22,33 +22,55 @@ import (
 // the projects on OperatorHub.io.
 //
 // This validator will ensure that:
+//
 // - The annotations capabilities into the CSV has a valid option, which are:
-// * "Basic Install"
-// * "Seamless Upgrades"
-// * "Full Lifecycle"
-// * "Deep Insights"
-// * "Auto Pilot"
```
Comment (ankitathomas) [inline pkg/validation/internal/operatorhub.go:30]: indentation also works for correcting the output formatting
```
// - The annotations capabilities into the CSV has a valid option, which are:
//   * "Basic Install"
//   * "Seamless Upgrades"
//   * "Full Lifecycle"
//   * "Deep Insights"
//   * "Auto Pilot"
```

**Review by ankitathomas — APPROVED** [review]
/lgtm

**Pull request merged**